nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/process-group-configuration.jsp

```diff
@@ -99,6 +99,17 @@
                             <span id="read-only-process-group-default-back-pressure-data-size-threshold" class="unset"></span>
                         </div>
                     </div>
+                    <div class="setting">
+                        <div class="setting-name">Log File Suffix
+                        <div class="fa fa-question-circle" alt="Info" title="Turns on dedicated logging. When left empty log messages will be logged only to the primary app log. When filled out messages logged by components in this group will be sent to the standard app log, as well as a separate log file, with the provided name, specific to this group."></div>
```

Comment:
Rather than "When filled out messages logged..." should maybe say "When set, messages logged..."